Default `ENABLE_LAPACK` as `OFF` (#825)
PennyLaneAI/pennylane-lightning: .github/CHANGELOG.md

### Improvements

> * Default `ENABLE_LAPACK` as `OFF` for `lightning` backends build.

Reviewer: ```suggestion
* `ENABLE_LAPACK` is off by default for all Lightning backends.
```
